example/generated-src/objc/TextSort-Bridging-Header.h

```diff
@@ -0,0 +1,7 @@
+// AUTOGENERATED FILE - DO NOT MODIFY!
```

Comment:
I remove the .m file on propose.
Since this is just a bridging header and the only function it's to import another files, I followed the same approach that Apple did.